src/modules/Selectors.jl

```diff
@@ -0,0 +1,151 @@
+"""
+An extensible code selection interface.
+
```

Comment:
Yes, an example of the generated code would be good, as well as the better description.